consul/leader.go

```diff
@@ -153,12 +153,6 @@ func (s *Server) establishLeadership() error {
 		return err
 	}
 
-	// Setup autopilot config if we are the leader and need to
```

Comment:
I think this endpoint will crash if the AP config hasn't been made yet:

https://github.com/hashicorp/consul/blob/master/consul/operator_autopilot_endpoint.go#L24-L30